editors/micro/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/zyedidia/micro 2.0.4 v
+
+homepage            https://micro-editor.github.io/
+
+description         A modern and intuitive terminal-based text editor
+
+long_description    micro is a terminal-based text editor that aims to be \
+                    easy to use and intuitive, while also taking advantage of \
+                    the capabilities of modern terminals. As its name \
+                    indicates, micro aims to be somewhat of a successor to \
+                    the nano editor by being easy to install and use. Micro \
+                    supports a full-blown plugin system. Plugins are written \
+                    in Lua and there is a plugin manager to automatically \
+                    download and install your plugins for you.
+
+categories          editors
+license             MIT
+
+build.target        github.com/go-delve/delve/cmd/dlv
```

Comment:
Not sure i get what this is for ?

Reply:
This was a leftover from when I first was using the `golang` portgroup's build command instead of invoking `make`.  Removed.